app/models/network_port.rb

```diff
@@ -55,7 +55,7 @@ def cloud_subnets_names
      allowed_address_pairs fixed_ips).each do |action|
 	  define_method("#{action}=") do |value|
       extra_attributes_save(action, value)
-    end
+   end
```

Comment:
GitHub is presenting this funny, but it's a hidden tab character being fixed to spaces.